manufacturing-ai-suite/wind-turbine-anomaly-detection/simulator/mqtt-publisher/publisher.py

```diff
@@ -72,43 +72,44 @@ def stream_csv(mqttc, topic, subsample, sampling_rate, filename):
     """
     Stream the csv file
     """
-    target_start_time = time.time()
-    row_count = 0
-    row_served = 0
-    jencoder = json.JSONEncoder()
-
-    print("\nMQTT Topic - {}\nSubsample - {}\n \
-          Sampling Rate - {}\nFilename - {}\n".format(topic,
-                                                      subsample,
-                                                      sampling_rate,
-                                                      filename))
+    loop = os.getenv("LOOP", "true").lower()
+    while True:
+        start_time = time.time()
+        row_served = 0
+        jencoder = json.JSONEncoder()
 
-    with open(filename, 'r') as fileobject:
-        tick = g_tick(float(subsample) / float(sampling_rate))
+        print(f"\nMQTT Topic - {topic}\nSubsample - {subsample}\nSampling Rate - {sampling_rate}\nFilename - {filename}\n")
```

Comment:
Can we have this print outside itself as this wont' change right?
Same holder true for line 79